modules/core/message_list_functions.php

```diff
@@ -351,13 +351,29 @@ function icon_callback($vals, $style, $output_mod) {
 if (!hm_exists('message_controls')) {
 function message_controls($output_mod) {
     $res = '<a class="toggle_link" href="#"><img alt="x" src="'.Hm_Image_Sources::$check.'" width="8" height="8" /></a>'.
-        '<div class="msg_controls">'.
-        '<a class="msg_read core_msg_control" href="#" data-action="read">'.$output_mod->trans('Read').'</a>'.
-        '<a class="msg_unread core_msg_control" href="#" data-action="unread">'.$output_mod->trans('Unread').'</a>'.
-        '<a class="msg_flag core_msg_control" href="#" data-action="flag">'.$output_mod->trans('Flag').'</a>'.
-        '<a class="msg_unflag core_msg_control" href="#" data-action="unflag">'.$output_mod->trans('Unflag').'</a>'.
-        '<a class="msg_delete core_msg_control" href="#" data-action="delete">'.$output_mod->trans('Delete').'</a>'.
-        '<a class="msg_archive core_msg_control" href="#" data-action="archive">'.$output_mod->trans('Archive').'</a>';
+        '<div class="msg_controls">
+            <div class="dynamic_dropdown on_mobile">
+              <button class="dynamic_dropdown-toggle" aria-haspopup="true" aria-expanded="false">
+                Actions
+                <span class="arrow"></span>
```

Comment:
This doesn't show the arrow on my side

Reply:
Sure ? can you test again please
<img width="432" alt="Capture d’écran 2024-01-03 à 11 49 46" src="https://github.com/cypht-org/cypht/assets/29623222/996e9ba8-9c4f-4af0-aaee-7936c5bcf8c7">

Reply:
Still not able to see the arrow

Reply:
I use a different theme from the default theme, that's why I couldn't see the arrow. When I switch to the default theme I see it. Can you try other themes and fix the error?